docs/understand/sizing-guide.md

```diff
@@ -139,7 +143,7 @@ where each commit contains **~180,000,000** objects (representing ~7.5 Petabytes
 All tests are reproducible using the [lakectl abuse command](../reference/commands.md#lakectl-abuse),
 so use it to properly size and tune your setup. All tests are accompanied by the relevant `lakectl abuse` command that generated them.
 
-### Random reads
+### Random reads - PostgresSQL
```

Comment:
Seems redundant stating its postgres or dynamodb, that's what we have the tabs for